app/models/cart.rb

```diff
@@ -32,4 +32,14 @@ def total
     end
   end
 
+  def discount_total
```

Comment:
Model method to determine a discounted sum. For some reason, it was very hard to update item_price in the view once a discount was applied, so this was a solution for the total method atm. I think when I add more BE logic for this problem, finding the total will become more intuitive.